keras/optimizer_v2/learning_rate_schedule.py

```diff
@@ -662,7 +662,7 @@ class CosineDecayRestarts(LearningRateSchedule):
   The learning rate multiplier first decays
   from 1 to `alpha` for `first_decay_steps` steps. Then, a warm
   restart is performed. Each new warm restart runs for `t_mul` times more
-  steps and with `m_mul` times smaller initial learning rate.
+  steps and with `m_mul` times larger initial learning rate.
```

Comment:
I think the smaller makes more correct impression here (this is a Decay, not a learning rate amplifier). Everytime when a warm start happens, the new learning rate will be existing learning rate times m_mul. Did I miss anything?

You can also take a look for https://towardsdatascience.com/https-medium-com-reina-wang-tw-stochastic-gradient-descent-with-restarts-5f511975163

Reply:
You are right in that you would typically want to have smaller learning rates on restarts. But mathematically, `m_mul` represents the inverse.

If you set `m_mul=2` you get a 2 times larger initial learning rate. 

If you set `m_mul=0.5` you get a 0.5 times larger (= 2 times smaller) initial learning rate.

Reply:
Sure. How about just remove the smaller/larger here since it is really depending the m_mul value here. "`m_mul` times initial learning rate" as the new learning rate seems reasonable to me as well.

Reply:
I agree, that removes the ambiguity quite nicely.